agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/ConfigurationTest.java

```diff
@@ -68,6 +69,15 @@ public void shouldParse() throws IOException {
         assertEquals(2, configuration.selfDiagnostics.file.maxHistory);
     }
 
+    @Test
+    public void shouldParseFromEnvVar() throws IOException {
+        envVars.set("APPLICATIONINSIGHTS_CONFIGURATION_CONTENT", "{\"connectionString\": \"InstrumentationKey=55555555-5555-5555-5555-555555555555\"}");
```

Comment:
does it support JMX metrics? can we add a test for it?

Reply:
added 👍